src/app/tests/suites/TestLevelControlWithOnOffDependency.yaml

```diff
@@ -0,0 +1,225 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Dependency with the On/Off cluster Verification (DUT as Server)
+
+config:
+    nodeId: 0x12344321
+    cluster: "Level Control"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Sends a MoveToLevel command to set current level to min value"
+      command: "MoveToLevel"
+      arguments:
+          values:
+              - name: "level"
+                value: 1
+              - name: "transitionTime"
+                value: 0
+              - name: "optionMask"
+                value: 0
```

Comment:
the move won't work here if the Onoff attribute is off. Set optionMask and optionOverride to 1 so it is done regardless of onoff state

Reply:
I will make the change. I was confused at first by the `Options` attribute. Thanks!

Reply:
Done.